index.rb

```diff
@@ -0,0 +1,10 @@
+require './person'
+require './capitalize'
```

Comment:
- [ ] **[OPTIONAL]** When importing your file, I would recommend you are mindful of the file names. Here you have `capitalize` whereas you should have `capitalizer`.

  ![pola-cap](https://user-images.githubusercontent.com/36542986/159520608-a29df7f3-6db6-4c5f-8339-876b33686e22.jpg)